### ISSUE TITLE
angleRange in HDrawablePool does not exist

### ISSUE THREAD

**Issue by danieltorrer** [hype/HYPE_Processing#147]
When opening demo HCircleLayout_007 I get the following error: `The function "angleRange(float, float) does not exist `. I checked in the javadoc reference and indeed there are not any functions with that name. 

Using latest version of HYPE and processing 3.3.6

**Comment by hype** [comment]
https://github.com/hype/HYPE_Processing/tree/lib_staging/examples/HCircleLayout

only has 6 examples ?

**Comment by danieltorrer** [comment]
True, however in the distribution .zip https://github.com/hype/HYPE_Processing/blob/lib_staging/distribution/HYPE.zip inside the folder `examples` there is a `_007` file.
